scalalib/api/src/mill/scalalib/api/ZincWorkerUtil.scala

```diff
@@ -50,6 +50,8 @@ trait ZincWorkerUtil {
   val NightlyVersion = raw"""(\d+)\.(\d+)\.(\d+)-bin-[a-f0-9]*""".r
   val TypelevelVersion = raw"""(\d+)\.(\d+)\.(\d+)-bin-typelevel.*""".r
 
+  private val FullVersion = """^([0-9]+)\.([0-9]+)\.([0-9]+)(-.*)?$""".r
```

Comment:
`ScalaJSFullVersion` and `ScalaNativeFullVersion` were exactly the same, and private, so I just combined them into this.